doc/htmldoc/index.rst

```diff
@@ -75,6 +83,36 @@ Welcome to the NEST Simulator documentation
 
              Discover all our examples!
 
+.. grid::
+
+    .. grid-item::
+        :columns: 6
+        :child-direction: row
+        :child-align: end
+
+        .. raw:: html
+
+           <object width="89%" height="89%" data="_static/img/excalidraw_nestconcept.svg" type="image/svg+xml"></object>
+
+
+    .. grid-item::
+        :columns: 5
+
+        **How NEST works - Conceptual approach**
+
+        This diagram provides an overview of the components essential for building and simulating a network model with NEST:
```

Comment:
I would drop this sentence entirely. I think it is all clear from the header and the following text.
```suggestion
```